framework/decode/vulkan_replay_dump_resources.cpp

```diff
@@ -1879,7 +1879,6 @@ VkResult VulkanReplayDumpResourcesBase::QueueSubmit(const std::vector<VkSubmitIn
                     GFXRECON_LOG_ERROR("Dumping draw calls failed (%s). Terminating.",
                                        util::ToString<VkResult>(res).c_str())
                     Release();
-                    exit(1);
```

Comment:
Removing these means that the message is no longer true.  But the message probably means that execution can't continue.  Doesn't this need to terminate somehow, either by `abort` or `throw` or some such?

Reply:
GFXRECON_LOG_FATAL?

Reply:
Looking at `framework/util/logging.h`, it seems there is no real difference between `GFXRECON_LOG_ERROR` and `GFXRECON_LOG_FATAL`. Do we want `FATAL` to terminate every time?